### PR TITLE
Euclid Algorithm

### PR DESCRIPTION
**Pull request by iam-sakshi**
<hr>

## Description 📜
Explain the brute force to find GCD and the Euclid way to find the GCD with explanation , code and Time Complexity. 
 
##Number Theory : Euclid Algorithm #7383
<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [x] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by iam-sakshi** [comment]
please review

**Comment by iam-sakshi** [comment]
@ravireddy07 please review

File: Competitive_Programming/C++/Maths & Number Theory/EuclidAlgorithm.md
Comment (ravireddy07) [inline Competitive_Programming/C++/Maths & Number Theory/EuclidAlgorithm.md:3]: Please differentiate this two approaches with explanation. Before going to code, the user/reader need to understand the concept/approach that used.

Like: how we're going to solve this problem in this approach.

File: Heap/TopKthFrequentElement/TopKthFrequentElement.md
Comment (ravireddy07) [inline Heap/TopKthFrequentElement/TopKthFrequentElement.md:1]: Remove  this file

**Review by ravireddy07 — CHANGES_REQUESTED** [review]

**Comment by iam-sakshi** [comment]
done all the required changes, please review

**Review by ravireddy07 — CHANGES_REQUESTED** [review]
Please differentiate this two approaches with explanation. Before going to code, the user/reader need to understand the concept/approach that used with Heading.

like Basic approach, Euclid Algorithm.

and place the formatted code.

**Comment by iam-sakshi** [comment]
I have placed the headings and the code is formatted . please review.

**Review by ravireddy07 — APPROVED** [review]

**Comment by iam-sakshi** [comment]
@Rishabh510 please review

**Comment by ravireddy07** [comment]
wait @Rishabh510 , Don't merge!

Seems like you have created another PR with same branch before merging this PR. Hence, you see there two files in this. @iam-sakshi .
Please follow the exact steps to create a PR.

**Comment by iam-sakshi** [comment]
I have deleted it. I am sorry ,I didn't noticed . 
But now there is only one file that was approved by @ravireddy07 .
@Rishabh510 please review

**Pull request merged**
